data/presets/shop/country_store.json

```diff
@@ -7,5 +7,8 @@
     "tags": {
         "shop": "country_store"
     },
-    "name": "Country Store"
+    "name": "Country Store",
+    "aliases": [
+        "Rural Supplies Store"
```

Comment:
I think this might be describing a farm suppy store?

Reply:
`country_store` is a somewhat UK centric tag and there is some overlap with other tags. Trying to reflect the wiki here:

- `country_store` - sells stuff aimed at country-dwellers and not necessarily farmers, may have stuff like outdoor clothing, stuff for riding, hunting, (big) gardens, pets, fuel, tools and other hardware. Wiki specifically mentions the term "rural supplies".
- `general` - rural/historic equivalent of a convenience or even small department store, may have similar items as above but more stuff for everyday needs (food etc)
- `agrarian` - equipment, tools etc. for farmers. May have similar items as the country store or even the general store, but most of all fertilizers, seeds, agricultural machines etc. 
- `garden center` - as above, but aimed at the (home) garden, so customers are not farmers but people with gardens
- `farm_supply` - finally, this tag is not documented in the wiki and barely used. There is probably little use for this tag because it really sounds like this is the same as `agrarian`

Reply:
Huh.  I don't get the difference between [shop=country_store ](https://wiki.openstreetmap.org/wiki/Tag:shop%3Dcountry_store) and [shop=agrarian](https://wiki.openstreetmap.org/wiki/Tag:shop%3Dagrarian).  From a US perspective, these seem like the same thing and I'd call it a farm supply store, farm store, agricultural supply store.  A country store is a different thing in my experience.  It's basically a rural general store including groceries and general household supplies.  Might have a few farm and garden things but that's definitely not the focus as described on the wiki.  Either this term has a very different meaning in UK English or the wiki is just wrong.

Reply:
Oops didn't see your reply with descriptions, @westnordost.  Seems like anything we'd call a country store in the US would just be `shop=general` and anything that should be tagged as `shop=country_store` we'd never call a country store 😄.

Reply:
`country stores` are aimed at country-dwellers with big gardens (and extravagant hobbies), `agrarian` are for farmers. Sure, there is some overlap, but the distinction is fair enough.
But anyway, what you are describing as "country store" for the US sounds like `shop=general`.

If the general understanding in en-US is that "country store" and "general store" are synonymous, then we'd need a name for `shop=country_store` that would fit the OSM wiki definition of that tag. The icon for thar preset (a cowboy hat) may be an indicator that it was also added with the wiki definition in mind.

Reply:
What would be more descriptive for a name then that is not in danger of confusion with `shop=general`?

Some ideas:

- country supplies store
- countryside supplies store
- rural supplies store
- cowboy wannabe store
- outdoor hobby store

Reply:
Honestly I'm not sure.  I theoretically get the distinction where `agrarian` is for stores caterering to working commercial farmers and `country_store` is more for stores catering to people who aren't in the commercial farming business but have small hobby farms, backyard chickens, large gardens, pets etc.  However, I've never really though of these as different types of stores.  They are just different variations of farm stores.  Maybe other en_US speakers make this distiction, but to me the idea of a farm store that isn't actually for farmers doesn't really make any sense.  I guess if I had to pick a name for them I just call them large farm store and small farm store 😀.

Reply:
Alright, I'll set the first alias as the primary name then because it seems to be slightly less confusing.